File: src/com/hkb48/keepdo/SortableListItem.java
package com.hkb48.keepdo;

import android.content.Context;
import android.util.AttributeSet;
import android.view.View;
import android.widget.LinearLayout;
import android.widget.TextView;

class SortableListItem extends LinearLayout {
    public SortableListItem(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    public void setText(String string) {
        TextView textView = (TextView) findViewById(R.id.textView1);
        textView.setText(string);
    }

    public View getGrabberView() {
        return findViewById(R.id.imageViewSort);
    }
}

File: app/src/main/java/com/hkb48/keepdo/SortableListItem.kt
package com.hkb48.keepdo

import android.content.Context
import android.util.AttributeSet
import android.view.View
import android.widget.LinearLayout
import com.hkb48.keepdo.databinding.TaskSortingListItemBinding

class SortableListItem(context: Context, attrs: AttributeSet?) : LinearLayout(context, attrs) {
    fun setText(string: String?) {
        TaskSortingListItemBinding.bind(this).textView1.text = string
    }

    val grabberView: View
        get() = TaskSortingListItemBinding.bind(this).imageViewSort
}